test/functionalTest/README.md

```diff
@@ -24,9 +24,8 @@ Then activate the virtual env:
 Next install accumulator-server.py depencencies:
 
 ```
-pip install Flask==1.0.2
-pip install pyOpenSSL==19.0.0
-pip install paho-mqtt==1.5.1
+pip install Flask
+pip install paho-mqtt
```

Comment:
I'd suggest to keep explicit versions in the pip install command. This makes installations much more controlled (otherwise a new version may appear of Flask that is not fully compatible with accumulator-server.py).

Why pyOpenSSL has been removed? I understand it's still needed as far as http support is required in accumulator-server.py